week-two/DoubleLinkedList.java

```diff
@@ -0,0 +1,76 @@
+public class DoubleLinkedList {
+  public DoubleNode head;
+
+  public DoubleLinkedList() {
+    this.head = null;
+  }
+
+  public void add(String z) {
```

Comment:
Avoid variable names like `z`, something like `value` would be better. Also, what if I wanted to add a set of numbers instead? 🤔 `Object` might be a better option to make this linkedlist more generic.